CMakeLists.txt

```diff
@@ -39,6 +39,7 @@ find_package(EXPAT REQUIRED)
 find_package(GLIB2 REQUIRED)
 find_package(GTHREAD2 REQUIRED)
 find_package(Libmagic REQUIRED)
+find_package(LIBMODULEMD REQUIRED)
```

Comment:
My only concern here is that libmodulemd is a required build dependency.
Some distros don't care about modularity and we should allow them to build without libmodulemd.

I recommend adding following option:
OPTION(WITH_MODULEMD "Enables modulemd support" ON)

Reply:
I discussed this with @dmach over IRC and we came to the conclusion that libmodulemd will be made optional, but that createrepo_c will refuse to operate on repos containing module metadata if it was not compiled with libmodulemd support. This is the only safe course of action, since inadvertently dropping module metadata can cause a great number of issues.

Reply:
This was implemented in the latest patches. Please re-review.